Build against latest release instead of pulling the latest
git version (which may fail from changes any time).

diff -u -r ../radare2-1.2.1/shlr/Makefile ./shlr/Makefile
--- ../radare2-1.2.1/shlr/Makefile	2017-02-05 11:37:02.000000000 +0100
+++ ./shlr/Makefile	2017-03-11 12:04:45.817180850 +0100
@@ -7,7 +7,7 @@
 include ../mk/${COMPILER}.mk
 
 # Build against last capstone release or next
-CS_RELEASE=0
+CS_RELEASE=1
 
 CAPSTONE_CFLAGS?=-g
 # CAPSTONE_MAKEFLAGS?=CAPSTONE_ARCHS="arm aarch64"
